recipe/meta.yaml

```diff
@@ -1,15 +1,15 @@
-{% set version = "2.37.0" %}
+{% set version = "2.42.0" %}
```

Comment:
Doing 2.42.0 rather than 2.43.0 because eccodes hasn't been build for 2.43.0 yet.

Reply:
Looking at https://confluence.ecmwf.int/display/ECC/Latest+news it isn't released yet, so good to go with 2.42.0. Thanks @xylar!

Reply:
This is odd because there's a tag for it:
https://github.com/ecmwf/eccodes/tags
and builds of the PyPI libraries:
https://pypi.org/project/eccodeslib/#files

Reply:
Yes, then probably the creation of the release was missed?